server/src/http/unleash_client.rs

```diff
@@ -188,7 +188,7 @@ impl UnleashClient {
         Self {
             urls: UnleashUrls::from_base_url(server_url),
             backing_client: new_reqwest_client(
-                "unleash_edge".into(),
+                "unleash_ede".into(),
```

Comment:
Not the place to make a change just to have tarpaulin run.

Reply:
I promise I'm not going to merge this!